velox/exec/StreamingAggregation.cpp

```diff
@@ -97,6 +118,17 @@ void StreamingAggregation::initialize() {
         rows_->rowSizeOffset());
   }
 
+  if (sortedAggregations_) {
```

Comment:
perhaps, extract all logic for initializing aggregate functions into a helper method

Reply:
Done